examples/adress/adress_protein/ubiquitin.py

```diff
@@ -0,0 +1,516 @@
+#!/usr/bin/python
```

Comment:
#!/usr/bin/env python2
?
Hardcoding the path to Python will mess up with virtualenvs, conda, etc